frontend/public/components/utils/_log-window.scss

```diff
@@ -24,9 +24,16 @@ $color-log-window-divider: $color-log-window-header-bg;
   padding-top: 10px;
 }
 
-.log-window__lines {
+.log-window__lines, .log-window__lines--wrap{
   padding-left: 10px;
   padding-right: 10px;
+}
+
+.log-window__lines {
   white-space: pre;
   width: 0;
```

Comment:
I don't think this existing declaration is needed, and it's actually causing a problem with my suggestion
```suggestion
```

Reply:
Made suggested changes. Resolving conversation